### DISCUSSION TITLE
Accessing CRS information from a LAZ/LAS file

### DISCUSSION THREAD

**Discussion by TarunSrinivas23** [laspy/laspy#278] [Q&A] [answered]
Hey Devs,
I am trying to use laspy to read a particular LAZ pointcloud to do some clipping using geopandas. I am facing issues trying to access the CRS data for this cloud. I downloaded this form the USGS website. How do I get to the CRS value after reading the file with laspy.read ?

This is how I am reading the LAZ file: 
```python
with laspy.open('/home/tarun/Downloads/USGS_LPC_MA_CentralEastern_2021_B21_19TCH324723.laz') as fh:
    print('Points from Header:', fh.header.point_count)
    las_file = fh.read()
    print('Points from data:', len(las_file.points))
print(las_file.vlrs))
```
Thanks in advance !

**Comment by tmontaigu** [comment, marked as answer]
You have to install pyproj and use [LasHeader.parse_crs()](https://laspy.readthedocs.io/en/latest/api/laspy.header.html#laspy.header.LasHeader.parse_crs)
